### DISCUSSION TITLE
Upload or download file to the plc

### DISCUSSION THREAD

**Discussion by hy1stax** [OpenLogics/MewtocolNet#16] [General]
Hello, guys, is you checked the command that could upload and download fpx file to target plc?

**Comment by Sandoun** [comment]
Hi, take a look at https://github.com/OpenLogics/MewtocolNet/blob/master/Examples/Examples.ProgramReadWrite/Program.cs

and https://github.com/OpenLogics/MewtocolNet/blob/1e2743603cd19e078a003cd0333ee91168c3c4a3/MewtocolNet/MewtocolInterfaceRequests.cs#L198

I think i did not write a upload method as of right now, also the upload download part is not checked in the main readme. Feel free to open a pull request for this feature.
